src/Countries/US/Ohio/OhioIncome/V20190101/OhioIncome.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Appleton\Taxes\Countries\US\Ohio\OhioIncome\V20190101;
+
+use Appleton\Taxes\Classes\Payroll;
+use Appleton\Taxes\Countries\US\Ohio\OhioIncome\OhioIncome as BaseOhioIncome;
+use Appleton\Taxes\Models\Countries\US\Ohio\OhioIncomeTaxInformation;
+use Illuminate\Database\Eloquent\Collection;
+
+class OhioIncome extends BaseOhioIncome
+{
+    const TAX_RATE = 0.05;
+
+    const TAX_WITHHOLDING_BRACKET = [
+        [0, .005, 0],
+        [5000, .01, 25],
+        [10000, .02, 75],
+        [15000, .025, 175],
+        [20000, .03, 300],
+        [40000, .035, 900],
+        [80000, .04, 2300],
+        [100000, .05, 3100],
+    ];
+
+    public function getTaxBrackets()
+    {
+        return [[0, self::TAX_RATE, 0]];
```

Comment:
wouldn't this return `TAX_WITHHOLDING_BRACKET`?

Reply:
Ya, I guess it does. I switched it to `return self::TAX_WITHHOLDING_BRACKET` and it produced the same results (Tests passing) 
How should I change it?

Reply:
`compute` is overwritten so it doesn't get called, just misleading to see a flat tax bracket there. You could make `getTaxAmountFromTaxBrackets` use `getTaxBrackets` instead of directly using `TAX_WITHHOLDING_BRACKET`. Not sure how far from the original implementation of `compute` it is then though